### PR TITLE
update deployer readme

### PR DESCRIPTION
**Pull request by jcantrill**
Update readme docs to deprecate the deployer and point people to use openshift-ansible

### CODE REVIEW

**Comment by ewolinetz** [comment]
I feel like we should also be pointing to a playbook that can be used and provide an example inventory file. We shouldn't just point to the role

**Comment by jcantrill** [comment]
@ewolinetz @richm  Updated to add a quickstart.  Thoughts?

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Following are instructions to standing up the logging stack.  Additional
+parameterization to further customize and alter the deployment can be found
+in the [openshift_logging role](https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging).  
+
+## Ansible Setup
+These instructions assume you are familiar with Ansible and have met the minimal [installation requirements](http://docs.ansible.com/ansible/intro_installation.html) for the control machine and managed nodes.
+
+1. `git clone https://github.com/ansible/ansible`
+2. `cd ansible && git checkout v2.2.0.0-1`
```
Comment (ewolinetz) [inline docs/quickstart.md]: i think we actually would need to use `2.2.1.0` now

Comment (jcantrill) [inline docs/quickstart.md]: Sum up my IRC comments:

```
FATAL: Current Ansible version (2.2.1.0) is not supported
```

File: docs/checking-efk-health.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+Check Fluentd logs for the message that it has read in its config file:
+```
+2016-02-19 20:40:44 +0000 [info]: reading config file path="/etc/fluent/fluent.conf"
+```
+
+Verify that Fluentd has been able to start reading in log files
+by checking the contents of `/var/log/node.log.pos` and `/var/log/es-containers.log.pos`.
+The `node.log.pos` file is a position file that keeps track of where Fluentd is in reading the syslog log files.  The `es-containers.log.pos` is a similiar file to keep track of where fluentd is in reading the docker log files (i.e `/var/log/containers`).  If the OpenShift node is configured to use the systemd journal as the log source, the position file is `/var/log/journal.pos`.
+
```
Comment (richm) [inline docs/checking-efk-health.md]: You can view the journal starting at the location where fluentd last read it by using a command like the following (as root):
```
# journalctl -c `cat /var/log/journal.pos`
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+3. `source hacking/env-setup`
+
+## Deploy Aggregated Logging
+
+1. Clone the Openshift installer
+```
+git clone https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_logging
```
Comment (richm) [inline docs/quickstart.md]: git clone https://github.com/openshift/openshift-ansible

**Comment by jcantrill** [comment]
@richm I made a few edits to bring the ansible code in line with master.  I believe we have a sufficient example inventory file that I tested against `oc cluster up` .  I'm going to merge based on your initial approval

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**